x-pack/plugins/cross_cluster_replication/public/app/sections/home/home.js

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { PureComponent, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { injectI18n, FormattedMessage } from '@kbn/i18n/react';
+import {
+  EuiPage,
+  EuiPageBody,
+  EuiPageContent,
+  EuiPageContentHeaderSection,
+  EuiTitle,
+  EuiSpacer,
+  EuiPageContentHeader,
+  EuiButton,
+} from '@elastic/eui';
+
+import routing from '../../services/routing';
+import { AutoFollowPatternList } from './auto_follow_pattern_list';
+import { BASE_PATH } from '../../../../common/constants';
+import { SectionUnauthorized } from '../../components';
+
+export class CrossClusterReplicationHomeUI extends PureComponent {
+  static propTypes = {
+    autoFollowPatterns: PropTypes.array,
+  }
+
+  state = {
+    sectionActive: 'auto-follow'
+  }
+
+  /**
+   * Get the button in top right corner. In phase 1 we only have the auto-follow patterns
+   * but in phase 2 it can be either "Add Follower" or "Add auto-follow pattern"
+   */
+  getButtonHeader() {
+    const { autoFollowPatterns } = this.props;
+    const { sectionActive } = this.state;
+
+    switch(sectionActive) {
+      case 'auto-follow': {
+        return autoFollowPatterns.length ? (
+          <EuiButton
+            {...routing.getRouterLinkProps(`${BASE_PATH}/auto_follow_patterns/add`)}
+            fill
+          >
+            <FormattedMessage
+              id="xpack.cross_cluster_replication.addAutoFollowPatternButtonLabel"
+              defaultMessage="Add auto-follow pattern"
+            />
+          </EuiButton>
+        ) : null;
+      }
+      default: {
+        // We will render here the "Create Follower" in Phase 2.
+        return null;
+      }
+    }
+  }
+
+  getHeaderSection() {
+    return (
+      <Fragment>
+        <EuiPageContentHeader>
+          <EuiPageContentHeaderSection>
+            <EuiTitle size="l">
+              <h1>
+                <FormattedMessage
+                  id="xpack.cross_cluster_replication.crossClusterReplicationTitle"
+                  defaultMessage="Cross Cluster Replication"
+                />
+              </h1>
+            </EuiTitle>
+          </EuiPageContentHeaderSection>
+          <EuiPageContentHeaderSection>
+            {this.getButtonHeader()}
+          </EuiPageContentHeaderSection>
+        </EuiPageContentHeader>
+        <EuiSpacer />
+      </Fragment>
+    );
+  }
+
+  /**
+   * In phase 1 we only have the aufo-follow patterns but in phase 2 we will add
+   * the follower indices. We will render here the Tab navigation at that moment.
+   * For now we simply render a Title
+   */
+  getTabsNav() {
```

Comment:
This is confusing for now as it does not return tabs but a title.  I know this will eventually return tabs, but maybe we can make the name make more sense for now as getTitle or some such and then change it when the tabs happen.

Reply:
Ok you're right, I will get rid of those "for the future" so it is clearer in the present 😊